src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-application-version.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Set application version
+type: apiDoc
+shortDescription: 'Set an application version to be associated with an interaction.'
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to set an application version to be associated with an interaction.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/withapplicationversion-android-agent-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/withapplicationversion-android-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/with-application-version
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-application-version-ios-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-application-version
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.withApplicationVersion(string $appVersion)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+NewRelic.withApplicationVersion(appVersion: String?)
+```
+
+## Description [#description]
+
+By default, the New Relic Android agent gets the app version from the `AndroidManifest.xml` or `build.gradle`. This method lets you override that value and set an app version by passing the app version. The app version is displayed as a dropdown in the New Relic UI, and you can use it to sort interactions.
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width={200}>
+        Parameter
+      </th>
+      <th width={200}>
+        Type
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$appVersion`
+      </td>
+
+            <td>
+        `string`
+      </td>
+
+      <td>
+        Required. String indicating the application version.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+
+## Example [#example]
+
+### Java [#java]
+
+```java
+NewRelic.withApplicationVersion('1.23.4')
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
```suggestion
```kotlin
NewRelic.withApplicationToken("").withApplicationVersion("1.23.4").start(applicationContext)

```
```